deploy_vcl

```diff
@@ -101,8 +106,7 @@ username = ENV['FASTLY_USER']
 password = ENV['FASTLY_PASS']
 
 if username.nil? or password.nil?
-  puts "Fastly credentials are not set. Exiting."
-  exit 1
+  raise 'Fastly credentials are not in the environment set as FASTLY_USER and FASTLY_PASS'
```

Comment:
I think thats a touch clunky.
if anything we should distinguish between them and have individual errors.
"FASTLY_PASS is not set as an environment variable"
"FASTLY_USER is not set as an environment variable"